test/test_utils.py

```diff
@@ -101,7 +106,12 @@ def is_named_tuple(nt):
     return all(type(name) == str for name in f)
 
 
-def assert_expected_wrapper(actual, expected):
+def assert_expected_wrapper(
```

Comment:
can we name this slightly more descriptive? maybe `assert_expected_named_tuple` or similar?